swagger.yaml

```diff
@@ -5822,6 +5822,16 @@ paths:
         - $ref: '#/components/parameters/UpdatedAtLTE'
         - $ref: '#/components/parameters/BackendUpdatedAtGTE'
         - $ref: '#/components/parameters/BackendUpdatedAtLTE'
+        - name: last_activity_lte
```

Comment:
should we insert this in here directly, or include it in the parameter section first?
it's currently only used once, to it won't make a difference